.circleci/config.yml

```diff
@@ -1,20 +1,44 @@
+defaults: &defaults
+  working_directory: /checkout
+  resource_class: large
+  steps:
+    - checkout
+    - setup_remote_docker:
+        reusable: true
+        exclusive: true
+    - run: make ci
+
+
 version: 2
 jobs:
-  build:
+  crossdock:
+    docker:
+      - image: golang:1.8.0
```

Comment:
Actually reading this - you can add this to defaults, and just use `golang:1.8.3` - we don't end up running our tests within these images, all testing is done using the remote docker daemon.